pkg/resourcecreator/resourcecreator.go

```diff
@@ -167,8 +167,15 @@ func Create(app *nais_io_v1alpha1.Application, resourceOptions ResourceOptions)
 				if err != nil {
 					return nil, fmt.Errorf("unable to assign sql password: %s", err)
 				}
-				sqlUser := GoogleSqlUser(app, instance, secretKeyRefEnvName, sqlInstance.CascadingDelete, resourceOptions.GoogleTeamProjectId)
+
+				sqlUser := GoogleSqlUser(app, instance, secretKeyRefEnvName, sqlInstance.CascadingDelete, resourceOptions.GoogleTeamProjectId, instance.Name)
```

Comment:
Better to merge the standard user with the additional users array, to have just one loop creating all the users.

Reply:
good ide, makes code cleaner